service/web/server_test.go

```diff
@@ -103,3 +103,49 @@ func TestGetProjectIDFromRequest(t *testing.T) {
 		assert.Equal(t, tst.err, err)
 	}
 }
+
+func Test_getResourceNameFromRequest(t *testing.T) {
```

Comment:
I believe this test case should be part of TestGetProjectIDFromRequest function only as that test is internally calling getResourceNameFromRequest(). Also we shouldn't write test case directly for private functions

Reply:
removed testcase for getResourceNameFromRequest and added in the existing testcase TestGetProjectIDFromReques